meetings/2023-04-06.md

```diff
@@ -0,0 +1,96 @@
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* SC
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* eP: JM, is there something we can look in the context of authz?
+* JM: Authn/z of webooks, was going to implement capability URLs. Technique to "kick the can down" the road for another way of doing authz. 
+* JM: There are implementations around doing ... how does a NotificationReceiver know that a notification came from a NotificationSender? As it is currently in CSS, the NS is generating a token using the IdP that comes with CSS. This is not a good solution for a few reasons. 1) we can't guarantee that a NS has access to an IdP. 2) Overly complicated as Solid-OIDC is designed around DPOP.
+* JM: eP made the case that a decent solution is capability URLs. cURLs does not require discussion over poken generation or authn. Creating an agreement between NS and NR. Probably more accurately the subscriber. In the future we might have authn scheme that generates tokens. Right now, the plan for implementation work to have capability URLs. That's where I am right now. eP can correct.
+* eP: Seems accurate. We can defer for unspecified for solving the problem using another authn scheme comparable to Solid OIDC. It is added convenience. Implementation doesn't need to be blocked until we resolve that topic.
+* eP: We use cURLs use. It is valid for streaming HTTP as well. There are other valid reasons ot have it to make explicit that it is used. For HTTP at least, receive cURL, so don't have tho do authn. It is already there - useful to have it explicit so that NR knows how to connect to the notification channel. There are some previous efforts to have authn directly on the endpoint over websocket protocol. it'd be beneficial if websocket channel doesn't have cURL but it can also use different authn scheme that's explicitly set for the channel.
```

Comment:
```suggestion
* eP: We use cURLs. They are valid for streaming HTTP as well. There are other valid reasons to make explicit that it is used. For HTTP at least, receive cURL, so don't have to do authn. It is already there — useful to have it explicit so that NR knows how to connect to the notification channel. There are some previous efforts to have authn directly on the endpoint over websocket protocol. It'd be beneficial if websocket channel doesn't have cURL but can use different authn scheme that's explicitly set for the channel.
```